lib/environment/i18n.js

```diff
@@ -56,3 +57,13 @@ export function i18n(messageName: string, ...substitutions: Array<string | numbe
 	// but Transifex does not create the `placeholders` field in exported JSON
 	// https://developer.chrome.com/extensions/i18n#examples-getMessage
 }
+
+export function i18nMarkdown(messageName: string, ...substitutions: Array<string | number>): string {
+	const message = i18n(messageName, ...substitutions);
+	// Some messages are not i18n-ized and may still be raw html
+	const isHTML = (/<[a-z][\s\S]*>/i).test(message);
+	if (process.env.NODE_ENV === 'development' && isHTML && messages.hasOwnProperty(messageName)) {
```

Comment:
Since this isn't very robust, this should probably be changed to always run through the markdown function when `message` is found in `messages`.